.github/workflows/5_builderpackage_engine-standalone.yml

```diff
@@ -0,0 +1,205 @@
+        path: src/external/rocksdb/build/
+
+    - name: Download wazuh shared libraries
+      uses: actions/download-artifact@v4
+      with:
+        name: wazuh-shared-libraries
+        path: src/
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: 3.12
+
+    - name: Install dependencies
+      run: |
+        ls -la src/build/engine
+        ls -la src/external/rocksdb/build
+        sudo apt-get update && sudo apt-get install -y libcap2-bin
+        pip install ${{env.ENGINE_DIR}}/tools/api-communication/
+        pip install ${{env.ENGINE_DIR}}/tools/engine-suite/
+        pip install ${{env.ENGINE_DIR}}/test/engine-test-utils/
+        pip install ${{env.ENGINE_DIR}}/test/helper_tests/engine-helper-test/
+
+    - name: Setup environment
+      run: |
+        chmod +x src/build/engine/wazuh-engine
+        python3 ${{env.ENGINE_DIR}}/test/setupEnvironment.py -e /tmp/actions
+
+    - name: Create zip scaffolding and copy files
+      run: |
+        install -d \
+          wazuh-engine-standalone/bin/lib \
+          wazuh-engine-standalone/default-security-policy \
+          wazuh-engine-standalone/data/store/schema \
+          wazuh-engine-standalone/data/kvdb \
+          wazuh-engine-standalone/data/tzdb \
+          wazuh-engine-standalone/logs \
+          wazuh-engine-standalone/sockets
+        touch wazuh-engine-standalone/bin/lib/.keep
+        touch wazuh-engine-standalone/default-security-policy/.keep
+        touch wazuh-engine-standalone/data/store/schema/.keep
+        touch wazuh-engine-standalone/data/kvdb/.keep
+        touch wazuh-engine-standalone/data/tzdb/.keep
+        touch wazuh-engine-standalone/logs/.keep
+        touch wazuh-engine-standalone/sockets/.keep
+        cp -r /tmp/actions/store/schema/* wazuh-engine-standalone/data/store/schema/
+        cp src/engine/standalone/run_engine.sh wazuh-engine-standalone/
+        chmod +x wazuh-engine-standalone/run_engine.sh
+        cp src/engine/standalone/README.md wazuh-engine-standalone/
+        cp src/external/rocksdb/build/librocksdb.so.8 wazuh-engine-standalone/bin/lib
+        cp src/libwazuhext.so wazuh-engine-standalone/bin/lib
+        cp src/build/engine/wazuh-engine wazuh-engine-standalone/bin/
+
+    - name: Compress standalone directory
+      run: |
+        zip -r wazuh-engine-standalone.zip wazuh-engine-standalone
+
+    - name: Set version
+      run: |
+        VERSION="$(grep '"version"' $GITHUB_WORKSPACE/VERSION.json | sed -E 's/.*"version": *"([^"]+)".*/\1/')"
+        echo "TAG=$VERSION" >> $GITHUB_ENV;
+
+    - name: Upload compressed zip file
+      uses: actions/upload-artifact@v4
+      with:
+        name: wazuh-engine-${{ env.TAG }}-standalone
+        path: wazuh-engine-standalone.zip
+
+  check_standalone_engine:
+    name: Test standalone engine
+
+    # Only runs if the PR status is different to Draft
+    if: ${{ !github.event.pull_request.draft }}
```

Comment:
This check is redundant, right? You've already defined the conditions for launching jobs above.